EXPLAINER.md

```diff
@@ -0,0 +1,227 @@
+            "rel": "contents"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneral.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBol.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBolIta.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralItalic.otf",
+            "encodingFormat": "font/otf"
+        }
+    ]
+}
+
+```
+
+Note that `readingOrder` defines the sequence of primary resources that form the publication. `resources` enumerates all the other resources that are required to render the publication. 
+
+## Design choices
+
+The unanswered question:
+
+> is the goal of this group to make a new packaging format for specialist book reading software and devices, or is it to obtain first class support for missing book-related features in the web platform as a whole?
+
+
+
+The key questions are [1] identifying the "bounds" of the publication, [2] defining the ordering of the primary resources, and [3] figuring out how to express metadata for the publication as a whole. 
+
+
+### 1. What's part of the publication?
+
+- The [Web Application Manifest spec](https://w3c.github.io/manifest/) uses `scope` to define the extent of a web application. 
+
+- Every component of an EPUB publication must be listed in an XML package file.
+
+- Web sites do not explicitly define their boundaries.
+
+- XML sometimes uses the idea of *transclusion*, where the contents of other documents can be incorporated into a parent document based on hypertext references. This concept can also be realized in HTML using iframes, HTML imports, custom elements, etc. The working group has shown no interest in this approach. 
+
+```html
+<html>
+  <head>
+    <title>Moby-Dick</title>
+  </head>
+  <body>
+    <web-publication>
+      <iframe src="c001.html">
+      <iframe src="c002.html">
+      <iframe src="c003.html">
+    </web-publication>
+  </body>
+</html>
+
+```
+
+The web publications spec has essentially adopted EPUB's approach, with an explicit list of resources. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. 
+
+### 2. Sequence of primary resources
+
+- HTML can express ordering of resources via `rel=prev` and `rel=next`, but these do not have UI support from the browsers (with notably rare exceptions).
+
+- HTML can also express an ordered list of links with the `nav` element. 
+
+- Transclusion can also define an ordering of the transcluded components. 
+
+- EPUB uses the `spine` element in the XML package file to determine a primary reading order.
+
+Once again, web publications use an explicit list outside of the content itself, via the `readingOrder` member of the manifest. 
+
```

Comment:
Do we want to also say that WP also has the notion of a TOC for a more detailed set of references (which may or may not be the same as the resources) and this is based on HTML structures (eg, `nav`)?

Reply:
We mention the TOC in the section on navigation. I'm adding a line to the manifest section saying we can refer to an HTML TOC there. See b789693.